### ISSUE TITLE
Upgrade to latest version of Discourse for Mozilla Learning instance

### ISSUE THREAD

**Issue by ldecoursy** [MozillaFoundation/mofo-devops#503]
### Details
We're currently running an outdated version of Discourse for https://forum.learning.mozilla.org. Can we update to v1.9?

**Comment by cadecairos** [comment]
@flamingspaz Would you be able to spare some cycles for this, or perhaps we can vidyo and you can show me the processes around deploying updates to the docker image? (It's on docker, iirc?)

**Comment by flamingspaz** [comment]
It does run on docker, but the whole image baking stuff is mostly manual iirc. It probably makes sense for me to make a Jenkins job that you can run whenever. I'll add it to my backlog but I can't guarantee I can get it done before the all hands. If I can't do it next week I can just update it manually.

**Comment by cadecairos** [comment]
Great, thanks! 

I don't think there's a rush on this unless there's an exploitable vulnerability that was patched between "v1.8.0.beta4 +12" and "1.9.0.beta1" ... do they publish security advisories?

**Comment by flamingspaz** [comment]
They do in the form of commits prefixed with `[security]`. @leomca has a thing to notify him, maybe he can push this into the #communityit IRC channel too?

**Comment by LeoMcA** [comment]
> I don't think there's a rush on this unless there's an exploitable vulnerability that was patched between "v1.8.0.beta4 +12" and "1.9.0.beta1"

There's probably something in there:
```
git log --pretty="%h %s" --no-merges v1.8.0.beta4...tests-passed | grep "SECURITY:"
19d5eb903c SECURITY: Vunerability in mail gem
e5e7a15a85 SECURITY: Never crawl by IP
908433a7a0 SECURITY: Validate the `entity` when downloading a CSV
58edd203a9 SECURITY: XSS issue in share popup if invalid link is passed in.
0013a23dc1 SECURITY: prefer render plain/html to render text where possible
cd39049262 SECURITY: do not send push notifications to suspended users
17f2974d0a SECURITY: Confirm new administrator accounts via email
3ef82bb32c SECURITY: CSRF vulnerabilities in `Admin::BackupsController`.
b49bf889f6 SECURITY: Disallow symlinks when restoring uploads.
1957d12670 SECURITY: Don't use backticks for exporting your archive
a690121805 SECURITY: always allow staff to resend activation mails
7ebfa3c901 SECURITY: Only allow users to resend activation email with a valid session.
76dd6933d2 Revert "Revert "Revert "SECURITY: Ensure oAuth authenticated email is the same as created user's email."""
e6d75f6844 Revert "Revert "SECURITY: Ensure oAuth authenticated email is the same as created user's email.""
0e3def7d2b Revert "SECURITY: Ensure oAuth authenticated email is the same as created user's email."
1060239e2d SECURITY: Ensure oAuth authenticated email is the same as created user's email.
0847b4258a Revert "SECURITY: Ensure that user has been authenticated."
fbe51d68a7 SECURITY: Ensure that user has been authenticated.
7a85469c4c SECURITY: inactive/suspended accounts should be banned from api
2dec731da3 SECURITY: correctly validate input when admin searches for screened ips
```

> They do in the form of commits prefixed with [security]. @LeoMcA has a thing to notify him, maybe he can push this into the #communityit IRC channel too?

It's a very simple IFTTT applet, so no. Slack, Telegram and any other service supported by IFTTT is possible.

I get it through email (and the IFTTT app) with this applet: https://ifttt.com/applets/QtjyYbmD-discourse-security-alerts

**Comment by cadecairos** [comment]
Well, discourse just went down, so I took the opportunity to ssh in there and rebuild the container. 

@ldecoursy we're now running the lastest version of Discourse (1.9.0.beta2)
